packages/newspringchurchapp/src/prayer/SaveButton/index.js

```diff
@@ -11,22 +12,27 @@ const SaveButtonConnected = memo(({ prayerID, saved, toggleSavedState }) => (
     {(save) => (
       <Mutation mutation={UNSAVE_PRAYER}>
         {(unSave) => (
-          <SaveButton
-            saved={saved}
-            onPress={() => {
-              toggleSavedState();
-              if (saved)
-                unSave({
-                  variables: { nodeId: prayerID },
-                  refetchQueries: [{ query: GET_SAVED_PRAYERS }],
-                });
-              else
-                save({
-                  variables: { nodeId: prayerID },
-                  refetchQueries: [{ query: GET_SAVED_PRAYERS }],
-                });
-            }}
-          />
+          <AnalyticsConsumer>
+            {({ track }) => (
+              <SaveButton
+                saved={saved}
+                onPress={() => {
+                  toggleSavedState();
+                  if (saved)
+                    unSave({
+                      variables: { nodeId: prayerID },
+                      refetchQueries: [{ query: GET_SAVED_PRAYERS }],
+                    });
+                  else
+                    save({
+                      variables: { nodeId: prayerID },
+                      refetchQueries: [{ query: GET_SAVED_PRAYERS }],
+                    });
+                  track({ eventName: saved ? 'Unsave' : 'Save' });
```

Comment:
This just feels clearer.
```suggestion
                  track({ eventName: saved ? 'Unsaved Prayer' : 'Saved Prayer' });
```